### ISSUE TITLE
Pull request #13 leads to Globalhashes being marked as unsuccessful

### ISSUE THREAD

**Issue by pfo** [relet/pygeohashing#16]
I like the idea behind pull request #13, Globalhashes should appear in their graticule's expedition list. Thank you for enhancing the bot, @piotrekoo!

However, the code seems to lead to Globalhashes being marked as unsuccessful (displaying the red `Arrow4.png`) in user lists. An example can be seen here: https://geohashing.site/?diff=890365

I've [addressed](https://geohashing.site/geohashing/User_talk:Piotrekoo) this on the wiki, but so far haven't gotten a response.

**Comment by aperfectring** [comment]
@pfo I can revert the changes from #13 if desired.  Let me know if that's what you'd like to happen.

**Comment by pfo** [comment]
@aperfectring Yes, this seems to be for the best since I am not aware for a fix for the issue.

**Comment by aperfectring** [comment]
Done and merged with #17
